src/common/components/organisms/EditorPanel.tsx

```diff
@@ -1,33 +1,86 @@
-import React, { useCallback } from "react";
-import { RiPencilFill } from "react-icons/ri";
+import React from "react";
 import { ThemeSettings } from "@/common/lib/theme";
-import ThemeEditorToolbar from "@/common/lib/theme/ThemeEditorToolbar";
+import ThemeSettingsEditor from "@/common/lib/theme/ThemeSettingsEditor";
 import DEFAULT_THEME from "@/common/lib/theme/defaultTheme";
-import { mergeClasses } from "@/common/lib/utils/mergeClasses";
+import FidgetTray from "./FidgetTray";
 
 export interface EditorPanelProps {
-  editMode: boolean;
   setEditMode: (editMode: boolean) => void;
   theme?: ThemeSettings;
   saveTheme: (newTheme: ThemeSettings) => void;
-  disabled?: boolean;
+  unselect: () => void;
+  selectedFidgetID: string | null;
+  currentFidgetSettings: React.JSX.Element;
```

Comment:
This should be `React.ReactNode` to allow it to properly be a `null` or `string` value in the event that no `currentFidget` is set